index.rst

```diff
@@ -24,20 +24,20 @@ Current Stable Server Release
 
 The administrator, user, and developer manuals for the current stable release are always at https ://doc.owncloud.org/server/latest/[manual].
 
----------------------------------------
-Latest Stable - Release ownCloud 10.0.1
----------------------------------------
-
-This documents the *latest production* release of ownCloud.
-
-* `User Manual <https://doc.owncloud.org/server/10.0/user_manual/>`_ (`Download
-  PDF <https://doc.owncloud.org/server/10.0/ownCloud_User_Manual.pdf>`_)
-* `Administration Manual <https://doc.owncloud.org/server/10.0/admin_manual/>`_
-  (`Download PDF
-  <https://doc.owncloud.org/server/10.0/ownCloud_Server_Administration_Manual.pdf>`_)
-* `Developer Manual <https://doc.owncloud.org/server/10.0/developer_manual/>`_
-  (`Download PDF
-  <https://doc.owncloud.org/server/10.0/ownCloudDeveloperManual.pdf>`_)
+----------------------------------------
+Latest Stable Release  - ownCloud 10.0.1
+----------------------------------------
+
+This documents the latest production version of ownCloud.
+
+* `User Manual <https://doc.owncloud.com/server/10.0/user_manual/>`_ (`Download 
```

Comment:
@settermjd This links now to the owncloud.com docs instead of the owncloud.org docs.